src/SDKs/Cdn/Management.Cdn/Microsoft.Azure.Management.Cdn.csproj

```diff
@@ -5,11 +5,11 @@
 
 	<PropertyGroup>
 		<Description>Microsoft Azure CDN Management Library</Description>
-		<Version>4.0.2-preview</Version>
+		<Version>4.0.3-preview</Version>
```

Comment:
you added a new functionality.
Update minor version, so make
nuget version = 4.1.2-preview
assemblyfileversion = 4.1.2

Reply:
Sure.